.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Need help? 
+Before open a new issue, did you ask your question on
+* [Google group](http://groups.google.com/group/testng-users)
+* [StackOverflow](http://stackoverflow.com/questions/tagged/testng)
+
+If you double post on both site, please provides the link to the other question to avoid duplicate answer.
```

Comment:
If post on both sites, please provide the link to the other question to avoid duplicating the answer.